app/controllers/user_sessions_controller.rb

```diff
@@ -90,11 +90,8 @@ def handle_social_login_flow(auth)
             flash[:notice] = "You are now following '#{params[:return_to].split('/')[4]}'."
             subscribe_multiple_tag(params[:return_to].split('/')[4])
             redirect_to '/dashboard', notice: "You have successfully signed in. Please change your password using the link sent to you via e-mail."
-          elsif params[:return_to] && params[:return_to] != "/signup" && params[:return_to] != "/login"
```

Comment:
Hi this we require due to infinite looping of /signup/login?return to=signup.....

Reply:
I think it's fine here since we are just redirecting to `/dashboard` without the `return_to` parameter

Reply:
I think we only need it if we do (what was happening before) `redirect to return_to + hash params` because hash params contains the `redirect_to` parameter. But if I redirect simple to `/dashboard` the return_to` parameters is not carried over.